securedrop_client/gui/widgets.py

```diff
@@ -197,12 +197,16 @@ def __init__(self) -> None:
         self.branding_barre.setPixmap(load_image("left_pane.svg"))
         self.user_profile = UserProfile()
 
+        # Add user guide links in left pane.
```

Comment:
I don't think it's necessary to say: "in left pane" since we're in the `LeftPane` class : )
Also, I suppose you meant "link" (singular)?